packages/app/lib/internal/registry/app.js

```diff
@@ -175,6 +177,19 @@ export function initializeApp(options = {}, configOrName) {
     .then(() => {
       app._initialized = true;
       return app;
+    })
+    .catch(e => {
+      // we need to clean the app entry from registry as the app does not actually exist
+      // There are still possible side effects from `onAppCreateFn` to consider but as existing
+      // code may rely on that function running prior to native create, re-ordering it is a semantic change
+      // and will be avoided
+      for (var key in APP_REGISTRY) {
+        if (key == 'bar') {
```

Comment:
👀 'bar'?

Reply:
Ah crap, obvious copy-paste error, also I realized while on a ride today it had an obvious test I should add to see it fail and then make sure apps length was still 1, which would have of course surfaced this junk immediately. Will correct